docs/diagrams/PlantUML File/StorageAddClientSequenceDiagram.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+hide footbox
+
+Participant "storage : Storage" as Storage #F4F4B6
+Participant "fw : FileWriter" as FileWriter #F4F4B6
+
+->Storage:addToClientFile(String: this.clientName, String: this.clientContactNumber, \nString: this.clientEmail, String: this.clientBudgetPerMonth)
+activate Storage #CDEFF9
+
+create FileWriter
+Storage -> FileWriter:FileWriter(String: CLIENT_PATH, boolean: append)
+activate FileWriter #CDEFF9
+return
+
+Storage -> FileWriter:write(String: newText)
+activate FileWriter #CDEFF9
+return
+
+Storage -> FileWriter:close()
+activate FileWriter #CDEFF9
+return
+destroy FileWriter
```

Comment:
Since it is difficult to make PlantUML destroy objects after showing a portion of a lifeline, you could add a note in the DG telling the user what the correct diagram should look like